### ISSUE TITLE
slideshow/code does not render the pipes for nontraditional symbols

### ISSUE THREAD

**Issue by rxg** [racket/racket#3453]
**What version of Racket are you using?**
7.8 [3m]

**What program did you run?**
Dr. Racket in the slideshow language

Please include a short example that triggers the bug
#lang slideshow
(require slideshow/code)

(slide
 #:title "Structure and Interpretation of Christian Parables"
 (code
  (define doubting-thomas-be-like '|¯\_(ツ)_/¯|)
  (define jesus-in-the-temple-be-like '|(╯°□°）╯︵ ┻━┻)|)
  (define ascii '|what in blazes?|)))

**What should have happened?**
The rendered slide should have looked like legal Racket code.  Instead it left off the pipes in all three cases, which is not legal Racket code.

**If you got an error message, please include it here.**

**Please include any other relevant details**
e.g., the operating system used or how you are running the code.

**Comment by mflatt** [comment]
Yes, this output is clearly wrong. I worry that fixing it will change existing uses of `code` that exploit current behavior intentionally or accidentally to some intended effect — but maybe that's a  worthwhile risk in this case.

**Comment by samth** [comment]
I definitely exploit this behavior intentionally in a bunch of my talks.

**Comment by rxg** [comment]
@samth In what way do you exploit this behaviour and why?  In my mind the default for code should be legal code, but I can imagine wanting other interesting effects when rendering.

**Comment by samth** [comment]
I've used this to typeset code that isn't Racket code (ie Java), where each line is just one symbol.

**Comment by rmculpepper** [comment]
I've used the existing behavior to typeset dots---for example, to explain that (a . (b . (c . ()))) = (a b c).

**Comment by mflatt** [comment]
The easy way out is a parameter to select between behaviors. And the backward-compatible way is to have the parameter default to the current behavior, but could we get away with backward incompatibility (with a relatively easy update for old uses) in this case?

**Comment by samth** [comment]
I guess I'm somewhat reluctant to have my old talks change, but I think an ok middle ground would be to have a warning printed about the situation.

**Comment by rxg** [comment]
> I've used the existing behavior to typeset dots---for example, to explain that (a . (b . (c . ()))) = (a b c).

I have definitely had situations where I would like to present a particular printed representation that may not be the default rendering, especially for teaching
